docs/api/context-bridge.md

```diff
@@ -46,6 +46,12 @@ The `contextBridge` module has the following methods:
 * `apiKey` string - The key to inject the API onto `window` with.  The API will be accessible on `window[apiKey]`.
 * `api` any - Your API, more information on what this API can be and how it works is available below.
 
+### `contextBridge.exposeInIsolatedWorld(worldId, apiKey, api)`
+
+* `worldId` Integer - The ID of the world to inject the API into. This has to be an existing world. `0` is the default world, `999` is the world used by Electron's `contextIsolation` feature. Using 999 would expose the object for preload context.
```

Comment:
Can we write a test to ensure no crash occurs when exposing an API in a world which doesn't exist yet?

Reply:
If it still works, maybe we can update these docs to remove the "existing world" part.

Reply:
There's a test that uses world id `1004`, resolving.